common/v2/features/SendAssets/stateFactory.tsx

```diff
@@ -194,7 +202,8 @@ const TxConfigFactory: TUseStateReducerFactory<State> = ({ state, setState }) =>
 
   const txFactoryState = {
     txConfig: state.txConfig,
-    txReceipt: state.txReceipt
+    txReceipt: state.txReceipt,
+    signedTx: state.signedTx
   };
```

Comment:
It's surprising that we can modify the return value without updating a TS declaration.

Reply:
Yeet.